spec/features/courses/fees_spec.rb

```diff
@@ -156,7 +156,8 @@
       end
 
       expect(course_fees_page.course_length_one_year).to_not be_checked
-      expect(course_fees_page.course_length_two_years).to be_checked
+      expect(course_fees_page.course_length_other).to be_checked
```

Comment:
expect(course_fees_page.course_length_two_years).to_not be_checked

can we add that line in please